common/sgx_workload/workload/ext_work_order_info_kme.cpp

```diff
@@ -249,8 +249,7 @@ int ExtWorkOrderInfoKME::CreateWorkOrderKeyInfo(
         public_sig_key = private_sig_key.GetPublicKey();
 
         ByteArray signing_key = StrToByteArray(private_sig_key.Serialize());
-        wo_key_info.verification_key = StrToByteArray(
-            public_sig_key.Serialize());
+        wo_key_info.verification_key = public_sig_key.Serialize();
```

Comment:
Please add a comment this is pem formatted string

Reply:
Done